docs/administering_lagoon/rbac.md

```diff
@@ -124,6 +124,7 @@ Here is a table that lists the roles and the access they have:
 | getProjectByEnvironmentId | project | view | projectID | Yes | Yes | Yes | Yes | Yes | Yes | Yes |  |
 | getProjectByGitUrl | project | view | projectID | Yes | Yes | Yes | Yes | Yes | Yes | Yes |  |
 | getProjectByName | project | view | projectID | Yes | Yes | Yes | Yes | Yes | Yes | Yes |  |
+| getAllProblems | problem | viewAll |  | Yes | Yes |  |  |  |  |  |  |
```

Comment:
@timclifford I'm not sure we need this extra permission.

I think the logic you have below that gets the user's project IDs are enough - that way the access to the projects (and their problems) are controlled via a single point (namely, the user's project list) rather than us now having this separate permissions path

Reply:
as discussed, removed this